avs-aggregator/compose.yml

```diff
@@ -16,10 +16,13 @@ services:
       - SUBSTRATE_RPC_URL=${SUBSTRATE_RPC_URL:-ws://collator-01:9944}
       - CHAIN_ID=31337
       - AVS_SERVER_IP_PORT_ADDRESS=0.0.0.0:8090
-      - AVS_BLOCK_VALIDATION_PERIOD=5
+      - AVS_BLOCK_VALIDATION_PERIOD=2
```

Comment:
is this ok for non e2e runs ? can we pass some specific envs just for e2e ?

Reply:
fixed!